<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusForm</class>
 <widget class="QWidget" name="ModbusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="RemoteIPAddress">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP Address:
172.22.0.28</string>
   </property>
  </widget>
  <widget class="QLabel" name="ThisPort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port: 50200</string>
   </property>
  </widget>
  <widget class="QLabel" name="thisLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>This</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemoteID">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unit ID: 0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectionButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect to remote</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateInfoButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ThisIPMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>25</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modbus Form</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoteIPMask">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="MIPAddress">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP Address:
172.22.0.28</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoteIDMask">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ThisID">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unit ID: 0</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>20</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>20</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ThisIDMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ThisPortMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>20</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="remoteLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote</string>
   </property>
  </widget>
  <widget class="QLabel" name="RemotePort">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port: 50200</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>20</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="RemotePortMask">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Statuslabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: Disconnected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveRightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <zorder>Title</zorder>
  <zorder>MIPAddress</zorder>
  <zorder>RemoteIPAddress</zorder>
  <zorder>ThisPort</zorder>
  <zorder>thisLabel</zorder>
  <zorder>RemoteID</zorder>
  <zorder>connectionButton</zorder>
  <zorder>UpdateInfoButton</zorder>
  <zorder>ThisIPMask</zorder>
  <zorder>testButton</zorder>
  <zorder>RemoteIPMask</zorder>
  <zorder>line_2</zorder>
  <zorder>RemoteIDMask</zorder>
  <zorder>ThisID</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line_7</zorder>
  <zorder>line_3</zorder>
  <zorder>ThisIDMask</zorder>
  <zorder>ThisPortMask</zorder>
  <zorder>line_6</zorder>
  <zorder>line_9</zorder>
  <zorder>remoteLabel</zorder>
  <zorder>RemotePort</zorder>
  <zorder>line_8</zorder>
  <zorder>RemotePortMask</zorder>
  <zorder>Statuslabel</zorder>
  <zorder>closeButton</zorder>
  <zorder>moveLeftButton</zorder>
  <zorder>moveRightButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
